harness/determined/pytorch/dsat/_dsat_search_method.py

```diff
@@ -9,24 +9,29 @@
 from abc import abstractmethod
 from collections import deque
 from dataclasses import dataclass
-from typing import Any, Dict, Iterable, Iterator, List, Optional, Set, Tuple, Union
+from typing import Any, Deque, Dict, Iterable, Iterator, List, Optional, Set, Tuple, Union, cast
 
 import numpy as np
 
-from determined import searcher
 from determined.experimental.client import create_experiment
 from determined.pytorch.dsat import _defaults, _utils
+from determined.searcher import (
```

Comment:
Maybe we should. I'll look into that, I did this because it made it easier to make sure that mypy behaved and shortened our code, as we had many type definitions that would use an entire line on its own

Reply:
yeah, I figured, and that makes total sense. I wasn't able to find any docs on confluence that actually dictated the above style guide, so maybe that's not actually the policy anymore.

I would just leave it for now until we can confirm.